### PR TITLE
add hint for log collect and k8s prometheus monitor in deploy doc 

### PR DESCRIPTION
**Pull request by yiduoyunQ**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### What is changed, added, or deleted? (Required)
add hint for log collect and k8s prometheus monitor in deploy doc 

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.6 (TiDB Operator 1.6 versions)
- [x] v1.5 (TiDB Operator 1.5 versions)
- [x] v1.4 (TiDB Operator 1.4 versions)
- [x] v1.3 (TiDB Operator 1.3 versions)
- [x] v1.2 (TiDB Operator 1.2 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

File: zh/deploy-on-aws-eks.md [hunk 1]
```diff
@@ -703,3 +703,15 @@ spec:
...
 
 更多可参考 [API 文档](https://github.com/pingcap/tidb-operator/blob/v1.6.0/docs/api-references/docs.md)和[集群配置文档](configure-a-tidb-cluster.md)完成 CR 文件配置。
+
+## 配置 TiDB 监控
+
+请参阅[部署 TiDB 集群监控与告警](monitor-a-tidb-cluster.md)。
+
+> **注意：**
+>
+> 建议[持久化监控数据](monitor-a-tidb-cluster#持久化监控数据)，并[设置 kube-prometheus 与 AlertManager](https://docs.pingcap.com/zh/tidb-in-kubernetes/stable/monitor-a-tidb-cluster#%E8%AE%BE%E7%BD%AE-kube-prometheus-%E4%B8%8E-alertmanager)。
```
[the remark this answers is not in the document]
Comment (yiduoyunQ) [reply zh/deploy-on-aws-eks.md]: fix md

[the remark this answers is not in the document]
Comment (yiduoyunQ) [reply zh/deploy-on-aws-eks.md]: # 链接我内部确认下需求

File: zh/deploy-on-aws-eks.md [hunk 2]
```diff
@@ -703,3 +703,15 @@ spec:
...
 
 更多可参考 [API 文档](https://github.com/pingcap/tidb-operator/blob/v1.6.0/docs/api-references/docs.md)和[集群配置文档](configure-a-tidb-cluster.md)完成 CR 文件配置。
+
+## 配置 TiDB 监控
+
+请参阅[部署 TiDB 集群监控与告警](monitor-a-tidb-cluster.md)。
+
+> **注意：**
+>
+> TiDB 监控默认不会持久化数据，建议[持久化监控数据](monitor-a-tidb-cluster.md#持久化监控数据)。 TiDB 监控不包含 POD 的 CPU、内存、磁盘监控，也没有报警系统，建议[设置 kube-prometheus 与 AlertManager](monitor-a-tidb-cluster.md#设置-kube-prometheus-与-AlertManager)。
```
Comment (Oreoxmt) [inline zh/deploy-on-aws-eks.md:713]:
```suggestion
> TiDB 监控默认不会持久化数据，建议[持久化监控数据](monitor-a-tidb-cluster.md#持久化监控数据)。 TiDB 监控不包含 POD 的 CPU、内存、磁盘监控，也没有报警系统，建议[设置 kube-prometheus 与 AlertManager](monitor-a-tidb-cluster.md#设置-kube-prometheus-与-alertmanager)。
```

File: zh/deploy-on-azure-aks.md [hunk 3]
```diff
@@ -569,3 +569,15 @@ Azure Disk 支持多种磁盘类型。若需要低延迟、高吞吐，可以选
...
 
     完成前面步骤后，local-volume-provisioner 即可发现集群内所有本地 NVMe SSD 盘。在 tidb-cluster.yaml 中添加 `tikv.storageClassName` 字段并设置为 `local-storage` 即可，可以参考前文[部署 TiDB 集群和监控](#部署-tidb-集群和监控)部分。
+
+## 配置 TiDB 监控
+
+请参阅[部署 TiDB 集群监控与告警](monitor-a-tidb-cluster.md)。
+
+> **注意：**
+>
+> TiDB 监控默认不会持久化数据，建议[持久化监控数据](monitor-a-tidb-cluster.md#持久化监控数据)。 TiDB 监控不包含 POD 的 CPU、内存、磁盘监控，也没有报警系统，建议[设置 kube-prometheus 与 AlertManager](monitor-a-tidb-cluster.md#设置-kube-prometheus-与-AlertManager)。
```
Comment (Oreoxmt) [inline zh/deploy-on-azure-aks.md:579]:
```suggestion
> TiDB 监控默认不会持久化数据，建议[持久化监控数据](monitor-a-tidb-cluster.md#持久化监控数据)。 TiDB 监控不包含 POD 的 CPU、内存、磁盘监控，也没有报警系统，建议[设置 kube-prometheus 与 AlertManager](monitor-a-tidb-cluster.md#设置-kube-prometheus-与-alertmanager)。
```

File: zh/deploy-on-gcp-gke.md [hunk 4]
```diff
@@ -448,3 +448,15 @@ spec:
...
 
 更多可参考 [API 文档](https://github.com/pingcap/tidb-operator/blob/v1.6.0/docs/api-references/docs.md)和[集群配置文档](configure-a-tidb-cluster.md)完成 CR 文件配置。
+
+## 配置 TiDB 监控
+
+请参阅[部署 TiDB 集群监控与告警](monitor-a-tidb-cluster.md)。
+
+> **注意：**
+>
+> TiDB 监控默认不会持久化数据，建议[持久化监控数据](monitor-a-tidb-cluster.md#持久化监控数据)。 TiDB 监控不包含 POD 的 CPU、内存、磁盘监控，也没有报警系统，建议[设置 kube-prometheus 与 AlertManager](monitor-a-tidb-cluster.md#设置-kube-prometheus-与-AlertManager)。
```
Comment (Oreoxmt) [inline zh/deploy-on-gcp-gke.md:458]:
```suggestion
> TiDB 监控默认不会持久化数据，建议[持久化监控数据](monitor-a-tidb-cluster.md#持久化监控数据)。 TiDB 监控不包含 POD 的 CPU、内存、磁盘监控，也没有报警系统，建议[设置 kube-prometheus 与 AlertManager](monitor-a-tidb-cluster.md#设置-kube-prometheus-与-alertmanager)。
```

**Review by csuzhangxc — APPROVED** [review]

File: zh/deploy-on-aws-eks.md [hunk 5]
```diff
@@ -703,3 +703,15 @@ spec:
...
 
 更多可参考 [API 文档](https://github.com/pingcap/tidb-operator/blob/v1.6.0/docs/api-references/docs.md)和[集群配置文档](configure-a-tidb-cluster.md)完成 CR 文件配置。
+
+## 配置 TiDB 监控
+
+请参阅[部署 TiDB 集群监控与告警](monitor-a-tidb-cluster.md)。
+
+> **注意：**
+>
+> TiDB 监控默认不会持久化数据，建议[持久化监控数据](monitor-a-tidb-cluster.md#持久化监控数据)。 TiDB 监控不包含 POD 的 CPU、内存、磁盘监控，也没有报警系统，建议[设置 kube-prometheus 与 AlertManager](monitor-a-tidb-cluster.md#设置-kube-prometheus-与-alertmanager)。
```
Comment (Oreoxmt) [inline zh/deploy-on-aws-eks.md:713]:
```suggestion
> TiDB 监控默认不会持久化数据，为确保数据长期可用，建议[持久化监控数据](monitor-a-tidb-cluster.md#持久化监控数据)。TiDB 监控不包含 Pod 的 CPU、内存、磁盘监控，也没有报警系统。为实现更全面的监控和告警，建议[设置 kube-prometheus 与 AlertManager](monitor-a-tidb-cluster.md#设置-kube-prometheus-与-alertmanager)。
```

File: zh/deploy-on-azure-aks.md [hunk 6]
```diff
@@ -569,3 +569,15 @@ Azure Disk 支持多种磁盘类型。若需要低延迟、高吞吐，可以选
...
 
     完成前面步骤后，local-volume-provisioner 即可发现集群内所有本地 NVMe SSD 盘。在 tidb-cluster.yaml 中添加 `tikv.storageClassName` 字段并设置为 `local-storage` 即可，可以参考前文[部署 TiDB 集群和监控](#部署-tidb-集群和监控)部分。
+
+## 配置 TiDB 监控
+
+请参阅[部署 TiDB 集群监控与告警](monitor-a-tidb-cluster.md)。
+
+> **注意：**
+>
+> TiDB 监控默认不会持久化数据，建议[持久化监控数据](monitor-a-tidb-cluster.md#持久化监控数据)。 TiDB 监控不包含 POD 的 CPU、内存、磁盘监控，也没有报警系统，建议[设置 kube-prometheus 与 AlertManager](monitor-a-tidb-cluster.md#设置-kube-prometheus-与-alertmanager)。
```
Comment (Oreoxmt) [inline zh/deploy-on-azure-aks.md:579]:
```suggestion
> TiDB 监控默认不会持久化数据，为确保数据长期可用，建议[持久化监控数据](monitor-a-tidb-cluster.md#持久化监控数据)。TiDB 监控不包含 Pod 的 CPU、内存、磁盘监控，也没有报警系统。为实现更全面的监控和告警，建议[设置 kube-prometheus 与 AlertManager](monitor-a-tidb-cluster.md#设置-kube-prometheus-与-alertmanager)。
```

File: zh/deploy-on-gcp-gke.md [hunk 7]
```diff
@@ -448,3 +448,15 @@ spec:
...
 
 更多可参考 [API 文档](https://github.com/pingcap/tidb-operator/blob/v1.6.0/docs/api-references/docs.md)和[集群配置文档](configure-a-tidb-cluster.md)完成 CR 文件配置。
+
+## 配置 TiDB 监控
+
+请参阅[部署 TiDB 集群监控与告警](monitor-a-tidb-cluster.md)。
+
+> **注意：**
+>
+> TiDB 监控默认不会持久化数据，建议[持久化监控数据](monitor-a-tidb-cluster.md#持久化监控数据)。 TiDB 监控不包含 POD 的 CPU、内存、磁盘监控，也没有报警系统，建议[设置 kube-prometheus 与 AlertManager](monitor-a-tidb-cluster.md#设置-kube-prometheus-与-alertmanager)。
```
Comment (Oreoxmt) [inline zh/deploy-on-gcp-gke.md:458]:
```suggestion
> TiDB 监控默认不会持久化数据，为确保数据长期可用，建议[持久化监控数据](monitor-a-tidb-cluster.md#持久化监控数据)。TiDB 监控不包含 Pod 的 CPU、内存、磁盘监控，也没有报警系统。为实现更全面的监控和告警，建议[设置 kube-prometheus 与 AlertManager](monitor-a-tidb-cluster.md#设置-kube-prometheus-与-alertmanager)。
```

File: zh/deploy-on-general-kubernetes.md [hunk 8]
```diff
@@ -133,3 +133,15 @@ aliases: ['/docs-cn/tidb-in-kubernetes/dev/deploy-on-general-kubernetes/','/zh/t
...
 >
 > TiDB（v4.0.2 起且发布于 2023 年 2 月 20 日前的版本）默认会定期收集使用情况信息，并将这些信息分享给 PingCAP 用于改善产品。若要了解所收集的信息详情及如何禁用该行为，请参见 [TiDB 遥测功能使用文档](https://docs.pingcap.com/zh/tidb/stable/telemetry)。自 2023 年 2 月 20 日起，新发布的 TiDB 版本默认不再收集使用情况信息分享给 PingCAP，参见 [TiDB 版本发布时间线](https://docs.pingcap.com/zh/tidb/stable/release-timeline)。
+
+## 配置 TiDB 监控
+
+请参阅[部署 TiDB 集群监控与告警](monitor-a-tidb-cluster.md)。
+
+> **注意：**
+>
+> TiDB 监控默认不会持久化数据，建议[持久化监控数据](monitor-a-tidb-cluster.md#持久化监控数据)。 TiDB 监控不包含 POD 的 CPU、内存、磁盘监控，也没有报警系统，建议[设置 kube-prometheus 与 AlertManager](monitor-a-tidb-cluster.md#设置-kube-prometheus-与-alertmanager)。
```
Comment (Oreoxmt) [inline zh/deploy-on-general-kubernetes.md:143]:
```suggestion
> TiDB 监控默认不会持久化数据，为确保数据长期可用，建议[持久化监控数据](monitor-a-tidb-cluster.md#持久化监控数据)。TiDB 监控不包含 Pod 的 CPU、内存、磁盘监控，也没有报警系统。为实现更全面的监控和告警，建议[设置 kube-prometheus 与 AlertManager](monitor-a-tidb-cluster.md#设置-kube-prometheus-与-alertmanager)。
```

File: zh/deploy-on-general-kubernetes.md [hunk 9]
```diff
@@ -133,3 +133,15 @@ aliases: ['/docs-cn/tidb-in-kubernetes/dev/deploy-on-general-kubernetes/','/zh/t
...
+
+## 日志收集
+
+系统与程序的运行日志对排查问题以及实现一些自动化操作可能非常有用，TiDB 各组件默认将日志输出在容器的 `stdout` 和 `stderr` 中，并依据容器运行时环境自动进行日志的滚动清理，当 POD 发生重启时容器日志会丢失，建议[收集 TiDB 及相关组件日志](logs-collection.md)。
```
Comment (Oreoxmt) [inline zh/deploy-on-general-kubernetes.md:147]:
```suggestion
系统与程序的运行日志对排查问题和实现自动化操作可能非常有用。TiDB 各组件默认将日志输出到容器的 `stdout` 和 `stderr` 中，并依据容器运行时环境自动进行日志的滚动清理。当 Pod 重启时，容器日志会丢失。为防止日志丢失，建议[收集 TiDB 及相关组件日志](logs-collection.md)。
```

File: zh/deploy-on-gcp-gke.md [hunk 10]
```diff
@@ -448,3 +448,15 @@ spec:
...
+
+## 日志收集
+
+系统与程序的运行日志对排查问题以及实现一些自动化操作可能非常有用，TiDB 各组件默认将日志输出在容器的 `stdout` 和 `stderr` 中，并依据容器运行时环境自动进行日志的滚动清理，当 POD 发生重启时容器日志会丢失，建议[收集 TiDB 及相关组件日志](logs-collection.md)。
```
Comment (Oreoxmt) [inline zh/deploy-on-gcp-gke.md:462]:
```suggestion
系统与程序的运行日志对排查问题和实现自动化操作可能非常有用。TiDB 各组件默认将日志输出到容器的 `stdout` 和 `stderr` 中，并依据容器运行时环境自动进行日志的滚动清理。当 Pod 重启时，容器日志会丢失。为防止日志丢失，建议[收集 TiDB 及相关组件日志](logs-collection.md)。
```

File: zh/deploy-on-azure-aks.md [hunk 11]
```diff
@@ -569,3 +569,15 @@ Azure Disk 支持多种磁盘类型。若需要低延迟、高吞吐，可以选
...
+
+## 日志收集
+
+系统与程序的运行日志对排查问题以及实现一些自动化操作可能非常有用，TiDB 各组件默认将日志输出在容器的 `stdout` 和 `stderr` 中，并依据容器运行时环境自动进行日志的滚动清理，当 POD 发生重启时容器日志会丢失，建议[收集 TiDB 及相关组件日志](logs-collection.md)。
```
Comment (Oreoxmt) [inline zh/deploy-on-azure-aks.md:583]:
```suggestion
系统与程序的运行日志对排查问题和实现自动化操作可能非常有用。TiDB 各组件默认将日志输出到容器的 `stdout` 和 `stderr` 中，并依据容器运行时环境自动进行日志的滚动清理。当 Pod 重启时，容器日志会丢失。为防止日志丢失，建议[收集 TiDB 及相关组件日志](logs-collection.md)。
```

File: zh/deploy-on-aws-eks.md [hunk 12]
```diff
@@ -703,3 +703,15 @@ spec:
...
+
+## 日志收集
+
+系统与程序的运行日志对排查问题以及实现一些自动化操作可能非常有用，TiDB 各组件默认将日志输出在容器的 `stdout` 和 `stderr` 中，并依据容器运行时环境自动进行日志的滚动清理，当 POD 发生重启时容器日志会丢失，建议[收集 TiDB 及相关组件日志](logs-collection.md)。
```
Comment (Oreoxmt) [inline zh/deploy-on-aws-eks.md:717]:
```suggestion
系统与程序的运行日志对排查问题和实现自动化操作可能非常有用。TiDB 各组件默认将日志输出到容器的 `stdout` 和 `stderr` 中，并依据容器运行时环境自动进行日志的滚动清理。当 Pod 重启时，容器日志会丢失。为防止日志丢失，建议[收集 TiDB 及相关组件日志](logs-collection.md)。
```

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**